site/docs/4.1/assets/scss/_brand.scss

```diff
@@ -10,62 +8,44 @@
   width: 100%;
   margin-bottom: 1rem;
   overflow: hidden;
-  color: #563d7c;
+  color: $bd-purple;
   background-color: #f9f9f9;
   border-radius: .25rem;
+
+  .inverse {
+    color: $white;
+    background-color: $bd-purple;
+  }
 }
 
 // Individual items
 .bd-brand-item {
   padding: 4rem 0;
   text-align: center;
-}
-.bd-brand-item + .bd-brand-item {
-  border-top: 1px solid #fff;
-}
-.bd-brand-logos .inverse {
-  color: #fff;
-  background-color: #563d7c;
-}
 
-// Heading content within
-.bd-brand-item h1,
-.bd-brand-item h3 {
-  margin-top: 0;
-  margin-bottom: 0;
-}
-.bd-brand-item .bd-booticon {
-  margin-right: auto;
-  margin-left: auto;
-}
+  + .bd-brand-item {
+    border-top: 1px solid $white;
+  }
 
-// Make the icons stand out on what is/isn't okay
-// .bd-brand-item .glyphicon {
-//   width: 30px;
-//   height: 30px;
-//   margin: 10px auto -10px;
-//   line-height: 30px;
-//   color: #fff;
-//   border-radius: 50%;
-// }
-// .bd-brand-item .glyphicon-ok {
-//   background-color: #5cb85c;
-// }
-// .bd-brand-item .glyphicon-remove {
-//   background-color: #d9534f;
-// }
+  // Heading content within
```

Comment:
There are no headings in `.bd-brand-item`, this can be removed.

Reply:
True, but note that I didn't remove all unused CSS. Just whatever I found while digging at it.